CMakeModules/ODMacroUtils.cmake

```diff
@@ -782,6 +782,24 @@ if ( WIN32 AND OD_MODULE_HAS_LIBRARY )
 
 endif()
 
+if ( APPLE AND OD_MODULE_HAS_LIBRARY )
+    if ( NOT "${OD_${OD_MODULE_NAME}_EXTERNAL_LIBS}" STREQUAL "" )
+        foreach(LIB ${OD_${OD_MODULE_NAME}_EXTERNAL_LIBS})
+			get_target_property(LIB_TYPE ${LIB} TYPE)
+			if(NOT LIB_TYPE STREQUAL "")
+				add_custom_command(TARGET ${OD_MODULE_NAME} POST_BUILD
+					COMMAND ${CMAKE_COMMAND} -E copy_if_different
+						"$<TARGET_SONAME_FILE:${LIB}>"
+						"$<TARGET_FILE_DIR:${OD_MODULE_NAME}>/"
+					COMMAND_EXPAND_LISTS
+					COMMENT "Copying ${LIB} symlink to Frameworks/Debug"
```

Comment:
Update comment: This is not a symbolic link, do not hard-code the output destination path:
"Copying runtime libraries of the plugin ${OD_MODULE_NAME}"